src/components/voyage/VoyageLogEntry.tsx

```diff
@@ -1,51 +1,54 @@
 import React from "react";
 import STTApi, { CONFIG } from "../../api";
-import { List, Image, Icon } from 'semantic-ui-react';
+import { Image, Icon } from 'semantic-ui-react';
 
 export class VoyageLogEntry extends React.Component<any,any> {
    constructor(props:any) {
       super(props);
 
       this.props.log.forEach((entry: any) => {
-         // TODO: some log entries have 2 crew
          if (entry.crew) {
-            let rc = STTApi.roster.find(rosterCrew => rosterCrew.symbol == entry.crew[0]);
-            if (rc) entry.crewIconUrl = rc.iconUrl;
+            entry.crewIconUrls = [];
+            for(let i = 0; i < entry.crew.length; i += 1) {
+               let rc = STTApi.roster.find(rosterCrew => rosterCrew.symbol == entry.crew[i]);
+               entry.crewIconUrls.push(rc ? rc.iconUrl ?? '' : '');
+            }
```

Comment:
This could be simpler as a one-liner via map:
entry.crewIconUrls = entry.crew.map(ec => STTApi.roster.find(rosterCrew => rosterCrew.symbol == ec)).map(rc => rc ? rc.iconUrl ?? '' : '');